packages/semantic-ui/src/PasswordWidget/PasswordWidget.js

```diff
@@ -23,7 +22,16 @@ function PasswordWidget(props) {
     uiSchema,
     formContext,
   } = props;
-  const semanticProps = getSemanticProps({ formContext, options });
+  const semanticProps = getSemanticProps({
+    schema,
+    uiSchema,
+    formContext,
+    options,
+    defaultSchemaProps: {
+      inverted: false,
+      fluid: true,
+    }
```

Comment:
You could also NOT pass in `defaultSchemaProps` since those two values are the default when not present